pbin.sh

```diff
@@ -56,6 +62,18 @@ pastebin() {
 		${expire+-F expire="$expire"} \
 		${reply+-F reply="$reply"} \
 		-F 'text=<-'
+else
+	# do paste
+	curl -s "$PASTE_URL${PASTE_APIKEY+?apikey=${PASTE_APIKEY}}" \
```

Comment:
lots of copy/paste with identical lines repeated. can't this be done more cleverly?